src/engine/v3/V3Interpreter.v3

```diff
@@ -1105,6 +1113,21 @@ component V3Interpreter {
 		}
 		push(Value.V128(low, high));
 	}
+	def doExtractLane<T, S>(size: byte, view: u64 -> T, extend: T -> S, stack_push: (S) -> void) {
```

Comment:
Can you pass in the log-base-2 of the size instead of the size, so that the divide and multiplies become shifts?

Reply:
Sure!